sources/PTYSession.m

```diff
@@ -3735,8 +3735,8 @@ - (void)changeFontSizeDirection:(int)dir
         [[ProfileModel sessionsInstance] setBookmark:[self profile] withGuid:guid];
 
         // Update an existing one-bookmark prefs dialog, if open.
-        if ([[[PreferencePanel sessionsInstance] window] isVisible]) {
-            [[PreferencePanel sessionsInstance] underlyingBookmarkDidChange];
+        if (PreferencePanel.sessionsInstance.windowIfLoaded.isVisible) {
```

Comment:
I'd prefer not to introduce stylistic inconsistency here; class methods are invoked with square brackets in the rest of the code.